docs/UserGuide.md

```diff
@@ -118,6 +118,54 @@ Showed all workouts in list
 ```
 ---
 
+### Update a Workout: `workout /update`
+Modifies an existing workout from the workout list.<br>
+*Noted: Only the number of repetition can be changed.*
+
+Format: `workout /update <index of workout to update> <new number of repetitions>`
+
+| Parameters                     | Description                                              |
+|--------------------------------|----------------------------------------------------------|
+| `<index of workout to update>` | The index of workout to be update from the workout list. |
```

Comment:
Minor language issue - "The index of workout to be updated..."

Reply:
Didn't notice that. Thank you for pointing it out.